RealmSwift/Realm.swift

```diff
@@ -116,8 +116,15 @@ public final class Realm {
      - throws: An `NSError` if the transaction could not be completed successfully.
                If `block` throws, the function throws the propagated `ErrorType` instead.
      */
-    public func write(_ block: () -> Void) throws {
-        try rlmRealm.transaction(block)
+    public func write(_ block: (() throws -> Void)) throws {
```

Comment:
No. `rethrows` doesn't apply since the function can throw even if the block doesn't.